client/components/container/container-config-directive.html

```diff
@@ -23,7 +23,10 @@
   </div>
 
   <div class="pk-sidebar-item">
-    <div class="pk-sidebar-item-label">header</div>
+    <div class="pk-sidebar-item-label">
+      header (supports
+      <a href="http://showdownjs.github.io/demo/" target="_blank">Showdown</a>)
```

Comment:
"supports Showdown" is not very helpful for people who don't recognize that this is a Markdown interpreter. Consider changing this to be clearer, for example:

header text (supports Showdown) => header (Showdown formatted text)

If I understand it right, the formatting gets applied unconditionally? This may cause issues where existing text gets reinterpreted as Showdown formatting directives. Not necessarily a big deal, but if there's no opt-out mechanism this should be explicit that content will always be interpreted this way.